pkg/util/encryption/multi_test.go

```diff
@@ -46,15 +47,16 @@ func TestOpen(t *testing.T) {
 				firstOpener.EXPECT().Open(mockInput).Return(nil, errors.New("fake error from the first opener"))
 				secondOpener.EXPECT().Open(mockInput).Return(nil, errors.New("fake error from the second opener"))
 			},
-			wantErr: "fake error from the second opener",
+			wantErr: "no openers succeeded:\n\t* first: fake error from the first opener\n\t* second: fake error from the second opener",
```

Comment:
So this is where we could use `errors.Is` for better tests

Reply:
rebased this PR without this commit